neural_compressor/strategy/strategy.py

```diff
@@ -591,6 +594,8 @@ def _initialize_recipe(self):
         logger.info(f"Adaptor has {len(adaptor_recipes)} recipes.")
         logger.debug(adaptor_recipes)
         usr_recipes_cfg = self.config.recipes if self.config.recipes else {}
+        # if smooth quant  is `True`, early stop

```

Comment:
Two spaces. # if smooth quant is `True`, early stop

Reply:
Thanks, fixed in [dbda02d](https://github.com/intel/neural-compressor/pull/1542/commits/dbda02d4da627fc624134cd4a18d4da60954d93d).